[CS2113T-F12-3] Teaching Assistant's Assistant

hozhenhong99 opened the pull request:
Teaching Assistant's Assistant helps teaching assistants manage details of their students. It is optimised for CLI users so that managing information can be done faster by typing in commands.

Rakesh12000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+  * Contains a list of `Module` objects currently being managed by the app.
+* `Util`
+  * Contains useful methods (e.g. Check if a string is integer/double, convert string to integer/double).
+* `Storage`
```
Youd could add UML diagrams for the Storage function. This will enable your users to understand how data is written and stored by TAA as well as how it is accessed an read as inputs.

Rakesh12000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+
+<br>
+
+### Add Assessment
```
The description for the implementation of Assessment tends to be a bit wordy and hard to follow. To make it easier for your users to understand this command function you could add an UML sequence diagram like the other commands.

conradwee replied:
Neatness/correctness:
looks messy
not well-formatted
hard to read/understand

Rakesh12000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+
+<br>
+
+### Set Marks
```
You could add an example scenario like the other commands so that it is easier for your users to understand how this command works.

conradwee replied:
Use of examples:

Not enough or too many examples e.g., sample inputs/outputs

Rakesh12000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+  * Defines how a command is to be executed.
+* `ModuleList`
+  * Contains a list of `Module` objects currently being managed by the app.
+* `Util`
```
You could further explain in-depth how the util component works. This will help your users get an idea of what util can be used for when they work on your code.

Rakesh12000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+9. A message will then be printed out to indicate to the user that the marks have been set successfully.
+
+
+### Set Attendance
```
You could use an UML sequence diagram here as well for easier reference of how this command function works for your users.

Rakesh12000 reviewed (commented):
DG Review

BryanElmer commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+* `StudentList#findStudent(keyword:String)` - Returns an ArrayList of students containing the keyword
+
+The sequence diagram shown below illustrates how the `add_student` command works:
+![AddStudentSequenceDiagram](diagrams/AddStudentSequenceDiagram.png)
```
You could maybe add numbering in the diagram so that you can link the steps to the diagram to make it more clear.

tttyyzzz replied:
Perhaps you could explain step by step how the add_student command is being parsed and saved.

tttyyzzz replied:
You could perhaps split up the uml diagram to include sub parts of the e.g. focuses on the logic only so that the uml diagram could be bigger and clearer. ref: https://se-education.org/addressbook-level3/DeveloperGuide.html

BryanElmer commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+## Implementation
+### Add Module
+The sequence diagram shown below illustrates how the `add_module` command works:
+![AddModuleSequenceDiagram](diagrams/AddModuleSequenceDiagram.png)
```
Nice description for the steps, but maybe you can add numbering in the diagram and link it to the steps.

BryanElmer commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+
+### Set Marks
+The sequence diagram shown below illustrates how the `set_mark` command works:
+![SetMarkSequenceDiagram](diagrams/SetMarkSequenceDiagram.png)
```
You could maybe also add numbering for the diagram so that readers can easily refer to the diagram from the steps.

BryanElmer commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+* `AttendanceList#sortAttendances` - Sorts the attendance in the `attendances` ArrayList in ascending order based on 
+lesson number.
+
+Below is an example scenario of how the set attendance feature behaves at each step:
```
The steps here look a bit chunky, maybe you could split them up into more steps which could help readers digest it better.

conradwee commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
 # Developer Guide
```
great work!

conradwee commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
-
+## Design
+### Architecture
+![ArchitectureDiagram](diagrams/ArchitectureDiagram.png)
```
would it be better to use UML here?

conradwee commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+lesson number.
+
+Below is an example scenario of how the set attendance feature behaves at each step:
+* Step 1 - The user executes `set_attendance c/CS2113T s/1 l/1 p/1` to set an attendance to `Present` for student at 
```
Neatness/correctness:
looks messy
not well-formatted
hard to read/understand

perhaps some more spacing at a minimum

conradwee reviewed (commented):
this looks like a great job, just make some minor changes and alls good

Nirmaleshwar commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
-
-* *glossary item* - Definition
-
-## Instructions for manual testing
-
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
```
I'm not sure if you're going you add this later but, perhaps adding some simple manual testing directions can guide the person to test functions you've described so far.

Nirmaleshwar commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+* `AttendnaceList#getAttendnaceIndex(lessonNumInput:String)` - Returns the attendance index in the `attendances` 
+ArrayList.
+* `AttendanceList#deleteAttendance(lessonNumInput:String)` - Deletes an attendance with a particular lesson number.
+* `AttendanceList#sortAttendances` - Sorts the attendance in the `attendances` ArrayList in ascending order based on 
```
Maybe can consider putting these on a table as they are a bit too hard to read.

Nirmaleshwar reviewed (commented):
Overall a very good job, just a few minor changes but since it's a work in progress, it's well on track.

peng-217 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+  * On app launch: Creates and runs an instance of `Taa`.
+* `UI`
+  * Handles UI operations.
+* `Taa`
```
The sequence diagram of TAA has a problem:
 
![image](https://user-images.githubusercontent.com/54028361/139199080-fcded7bb-e20f-46a4-861e-4119e5cf6c1e.png)

The activation bar of Taa should be closed

peng-217 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+  * On run: Loads persistent data from `Storage`, receives user input from `UI`, and uses `Parser` to parse the user input.
+* `Parser`
+  * Handles input parsing and determines which command to run.
+* `Command`
```
Would it be better to add function notation?

![image](https://user-images.githubusercontent.com/54028361/139200528-bb709181-5d51-4309-8ea5-4938a749479f.png)

peng-217 replied:
Better to add function notation
![image](https://user-images.githubusercontent.com/54028361/139203103-49e3484c-b8b0-4697-b804-28ec33f4deb1.png)

peng-217 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,228 @@
+<br>
+
+**Interaction between components**
+![ArchitectureSequenceDiagram](diagrams/ArchitectureSequenceDiagram.png)
```
It would be better to add the notation of the grey file icon
![image](https://user-images.githubusercontent.com/54028361/139202158-c53e72f1-a6ff-4ede-90cb-86e446252b1b.png)

tttyyzzz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,231 @@
+* `Util`
+  * Contains useful methods (e.g. Check if a string is integer/double, convert string to integer/double).
+* `Storage`
+  * Handles data storage operations (e.g. Reading from and writing to data file).
```
![image](https://user-images.githubusercontent.com/21839618/139384432-1ac7e4eb-0138-4143-a908-188bb805f4b9.png)
Perhaps adding the new ModuleList()

tttyyzzz replied:
![image](https://user-images.githubusercontent.com/21839618/139384561-aca95cc7-4e8d-4dab-8810-d61b9d9d7689.png)
Perhaps adding new Command() upon instantiation

tttyyzzz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,231 @@
+  from the `assessments` ArrayList.
+
+The sequence diagram shown below illustrates how the `add_assessment` command works:
+![AddAssessmentSequenceDiagram](diagrams/AddAssessmentSequenceDiagram.png)
```
![image](https://user-images.githubusercontent.com/21839618/139385372-936204b7-2394-4fb7-93b7-cace8cc63d7f.png)
Perhaps adding the 'X' below the addModuleCommand

tttyyzzz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,231 @@
+## Implementation
+### Add Module
+The sequence diagram shown below illustrates how the `add_module` command works:
+![AddModuleSequenceDiagram](diagrams/AddModuleSequenceDiagram.png)
```
![image](https://user-images.githubusercontent.com/21839618/139385859-87127db3-7a35-4aad-87ad-d29d5b605a25.png)
Perhaps deleting the redundant class.

tttyyzzz reviewed (commented):
Overall good uml diagrams with detailed labels. Could use numbering in uml diagrams if it helps in explanation. Perhaps standardise the steps underneath the uml diagrams. Good Job!

rachelkeh committed:
Change type of assessment from int to double and update all the string formatting accordingly

hozhenhong99 committed:
update PPP

hozhenhong99 committed:
Merge pull request #244 from rachelkeh/branch-assessment

Change type of assessment from int to double and update all the strin…

leyondlee committed:
Merge pull request #245 from hozhenhong99/student-branch

update PPP

leyondlee committed:
Add PPP

leyondlee committed:
Minor wording changes

leyondlee committed:
Update TaaClassDiagram and CommandClassDiagram

leyondlee committed:
Change maximumMarks in AssessmentDeserializer to double

leyondlee committed:
Check if student comment is valid in StudentDeserializer

leyondlee committed:
Simplify verify() method in TeachingClass

hozhenhong99 committed:
add images to UG

rachelkeh committed:
Update comparison of doubles to a higher precision with Double.compare

hozhenhong99 committed:
Merge pull request #246 from leyondlee/branch-ppp

Add PPP

hozhenhong99 committed:
Merge pull request #247 from leyondlee/branch-dg

Update TaaClassDiagram and CommandClassDiagram

hozhenhong99 committed:
Merge pull request #248 from leyondlee/branch-class

Change maximumMarks in AssessmentDeserializer to double and check if student comment is valid in StudentDeserializer

hozhenhong99 committed:
Merge branch 'master' of http://github.com/AY2122S1-CS2113T-F12-3/tp into student-branch

hozhenhong99 committed:
update PPP

rachelkeh committed:
Merge branch 'master' into branch-assessment

rachelkeh committed:
Merge pull request #251 from hozhenhong99/student-branch

add images to UG

hozhenhong99 committed:
Merge pull request #250 from rachelkeh/branch-assessment

Update comparison of doubles to a higher precision with Double.compare

rachelkeh committed:
Fix logic error

hozhenhong99 committed:
update formatting for UG

rachelkeh committed:
Merge pull request #252 from hozhenhong99/testing-branch

update formatting for UG

leyondlee committed:
Merge pull request #253 from rachelkeh/branch-assessment

Fix logic error

hozhenhong99 committed:
add junit testing for add, edit and delete student

rachelkeh committed:
Merge pull request #254 from hozhenhong99/testing-branch

add junit testing for add, edit and delete student

hozhenhong99 committed:
add JUnit testing for set and delete comment

rachelkeh committed:
Merge pull request #255 from hozhenhong99/testing-branch

add JUnit testing for set and delete comment

rachelkeh committed:
Upload pictures for expected output and edit some errors in UG. Add note too

rachelkeh committed:
Edit grammar error in add assessment message

rachelkeh committed:
Update 2dp note for decimal values

hozhenhong99 committed:
fix formatting

leyondlee committed:
Add ability to stop Storage from writing/reading by setting filename to null

leyondlee committed:
Set Storage filename to null for tests

leyondlee committed:
Add more test cases for isStringInteger, isStringDouble, isStringBoolean

leyondlee committed:
Move common message to Command class

leyondlee committed:
Allow empty arguments for EditClassCommand and check for invalid values

leyondlee committed:
Add test cases for AddClassCommand, DeleteClassCommand, EditClassCommand

leyondlee committed:
Merge pull request #259 from hozhenhong99/testing-branch

fix formatting

leyondlee committed:
Merge pull request #258 from rachelkeh/branch-assessment

Edit grammar error in add assessment message

leyondlee committed:
Merge branch 'master' into branch-class

leyondlee committed:
Fix checkstyle

leyondlee committed:
Change Parser.splitFirstSpace method to private

leyondlee committed:
Add author tag

leyondlee committed:
Remove extra lines

leyondlee committed:
Change class ID to uppercase

leyondlee committed:
Update EXPECTED.txt

leyondlee committed:
Update EditClassCommandTest

leyondlee committed:
Add images for class commands

leyondlee committed:
Update AddClassCommandTest and EditClassCommandTest

hozhenhong99 committed:
Merge pull request #260 from leyondlee/branch-class

Set Storage filename to null for tests, allow empty arguments for EditClassCommand and check for invalid values, and add test cases for AddClassCommand, DeleteClassCommand, EditClassCommand

hozhenhong99 committed:
Merge pull request #261 from leyondlee/branch-ug

Add images for class commands in UG and fix spaces under images

hozhenhong99 committed:
update file name

daknam2001 committed:
add set_attendance for range of student indexes
add set_attendance for selection of student indexes
update user guide with examples

daknam2001 committed:
fix checkstyle issues

daknam2001 committed:
update usage message

daknam2001 committed:
fix checkstyle

leyondlee committed:
Change welcome message position

leyondlee committed:
SLAP Parser.getArgumentFromString method and add JavaDoc

leyondlee committed:
Fix formatting

leyondlee committed:
Remove redundant conditions

leyondlee committed:
Simplify archive command

leyondlee committed:
Fix formatting

leyondlee committed:
Fix formatting

leyondlee committed:
SLAP assessment deserializers

leyondlee committed:
SLAP attendance deserializers

leyondlee committed:
Fix formatting for StorageDeserializer

leyondlee committed:
SLAP student deserializers

leyondlee committed:
SLAP class deserializers

leyondlee committed:
Handle null JSON elements

leyondlee committed:
Update test cases

leyondlee committed:
Fix checkstyle

leyondlee committed:
Fix formatting

leyondlee committed:
Update archive command

leyondlee committed:
Fix logic error in Assessment.verify method

leyondlee committed:
Add AssessmentDeserializerTest

leyondlee committed:
Add AssessmentListDeserializerTest

hozhenhong99 committed:
Merge pull request #263 from daknam2001/multiple-set

add set_attendance for range of student indexes

jon-the-melon committed:
Merge pull request #262 from hozhenhong99/PPP-branch

update file name

hozhenhong99 committed:
Merge pull request #264 from leyondlee/branch-class

SLAP methods, handle null JSON elements, remove filename argument from archive command

hozhenhong99 committed:
Merge pull request #265 from leyondlee/branch-ug

Fix image formatting and update archive command in UG

rachelkeh committed:
Update to n/

rachelkeh committed:
Fix logic error in verify method

leyondlee committed:
Update assessment deserializer tests

leyondlee committed:
Add check for invalid json object

leyondlee committed:
Add tests for deserializers

leyondlee committed:
Merge pull request #266 from rachelkeh/branch-assessment

Fix logic error in verify method

leyondlee committed:
Merge branch 'master' into branch-class

# Conflicts:
#	src/main/java/taa/assessment/Assessment.java

leyondlee committed:
Add author tag

jon-the-melon committed:
fix escaping of pipe character in command summary and delete extraneous line after list_marks

leyondlee committed:
Fix checkstyle

leyondlee committed:
Fix checkstyle

jon-the-melon committed:
Merge pull request #267 from leyondlee/branch-class

Add tests for deserializers

jon-the-melon committed:
Merge pull request #268 from jon-the-melon/master

fix escaping of pipe character in command summary and delete extra line after list_marks

jon-the-melon committed:
Merge pull request #256 from rachelkeh/branch-UG

Upload pictures for expected output and edit some errors in UG. Add n…

jon-the-melon committed:
update ppp

jon-the-melon committed:
Merge pull request #269 from jon-the-melon/master

update ppp

leyondlee committed:
Add check for no. of decimal places

leyondlee committed:
Add newline to end of file

jon-the-melon committed:
Merge pull request #270 from leyondlee/branch-class

Add overload Util.isStringDouble method to check for no. of decimal places

leyondlee committed:
Add more tests for Util.isStringDouble

jon-the-melon committed:
add change assessment name key in hashmap for student class

jon-the-melon committed:
fix checkstyle

hozhenhong99 committed:
Merge pull request #289 from jon-the-melon/master

add change assessment name key in hashmap for student class

leyondlee committed:
Update PPP

leyondlee committed:
Update ArchitectureDiagram.puml

leyondlee committed:
Update ArchitectureSequenceDiagram to include exception

leyondlee committed:
Update AddAssessmentSequenceDiagram

leyondlee committed:
Update AddClassDiagram

leyondlee committed:
Update AddStudentSequenceDiagram

leyondlee committed:
Update SetMarkSequenceDiagram

leyondlee committed:
Fix wrong method call

leyondlee committed:
Change module methods to class

leyondlee committed:
Change module methods to class

leyondlee committed:
Update set_mark explanation

rachelkeh committed:
Update error message for weightage and marks

leyondlee committed:
Merge branch 'master' into branch-dg

leyondlee committed:
Merge branch 'master' into branch-class

rachelkeh committed:
Update error message for weightage and maximum marks in edit assessment

leyondlee committed:
Update DeleteCommentCommand usage

hozhenhong99 committed:
Merge pull request #291 from leyondlee/branch-dg

Update add_student, add_assessment, set_mark diagrams and include exception in ArchitectureSequenceDiagram

hozhenhong99 committed:
Merge pull request #290 from leyondlee/branch-class

Add more tests for Util.isStringDouble and update DeleteCommentCommand usage

hozhenhong99 committed:
update UG to display list_comments

leyondlee committed:
Fix formatting

leyondlee committed:
Merge pull request #293 from hozhenhong99/final-edits-branch

update UG to display list_comments

leyondlee committed:
Change "Abort" to "Aborted"

rachelkeh committed:
Include new upper bound for maximum marks

rachelkeh committed:
Fix checkstyle

jon-the-melon committed:
change images to reflect correct expected output in ug

jon-the-melon committed:
Merge pull request #294 from leyondlee/branch-class

Fix formatting and change "Abort" to "Aborted"

leyondlee committed:
Rename .PNG to .png

leyondlee committed:
Move common notes to top of category section

jon-the-melon committed:
edit UG to mention that marks input can have at most 2 decimalpoints

leyondlee committed:
Add images for archive and reset commands

leyondlee committed:
Merge pull request #295 from jon-the-melon/master

change images to reflect correct expected output in ug

leyondlee committed:
Merge pull request #296 from leyondlee/branch-ug

Move common notes to top of category section and add images for archive and reset commands in UG

leyondlee committed:
Remove spacing before horizontal line

leyondlee committed:
Add spacing before horizontal line

leyondlee committed:
Merge branch 'branch-ug' of github.com:leyondlee/tp into branch-ug

# Conflicts:
#	docs/UserGuide.md

leyondlee committed:
Add spacing before horizontal line

rachelkeh committed:
Update grammar errors

leyondlee committed:
Update README.md

hozhenhong99 committed:
change sort by scores to 6dp, add assertion for student and comment commands

rachelkeh committed:
Merge remote-tracking branch 'upstream/master' into branch-assessment

hozhenhong99 committed:
Merge pull request #297 from leyondlee/branch-ug

Fix formatting

hozhenhong99 committed:
Merge pull request #298 from leyondlee/branch-readme

Update README.md

leyondlee committed:
Merge pull request #299 from hozhenhong99/final-edits-branch

change sort by scores to 6dp, add assertion for student and comment c…

leyondlee committed:
Fix formatting

leyondlee committed:
Merge pull request #300 from leyondlee/branch-ug

Fix formatting

leyondlee committed:
Update user stories

leyondlee committed:
Update AboutUs.md

hozhenhong99 committed:
Merge pull request #301 from leyondlee/branch-dg

Update user stories in DG and AboutUs.md

hozhenhong99 committed:
update PPP

leyondlee committed:
Merge pull request #302 from hozhenhong99/final-edits-branch

update PPP

hozhenhong99 committed:
update PPP formatting

rachelkeh committed:
Update edit assessment command to change the results hashmap key from the old assessment name to the new assessment name

hozhenhong99 committed:
Merge pull request #303 from hozhenhong99/final-edits-branch

update PPP formatting

rachelkeh committed:
Merge branch 'master' into branch-assessment

rachelkeh committed:
Update error message and change to isStringDouble(maximumMarksString, 2) to accomodate only 2 dp in maximum marks and weightage

rachelkeh committed:
Update tests

leyondlee committed:
Fix formatting

leyondlee committed:
Add template PPP for Rachel and Dak Nam

rachelkeh committed:
Update edit assessment to check whether the new maximum marks exceed the current marks of any students and if so, count the number of exceeds and print it

rachelkeh committed:
Update checkstyle

rachelkeh committed:
Update UG to reflect the new 2 dp restriction and justification for why maximum marks and weightage can be 0

rachelkeh committed:
Merge branch 'master' into branch-UG

rachelkeh committed:
Merge pull request #304 from leyondlee/branch-ppp

Add template PPP for Rachel and Dak Nam

daknam2001 committed:
add range and selection for delete attendance command
add limit for lessons (1000)
update format for help function (attendance)
update test cases
add expected output images for user guide
update sequence diagram
update developer guide

daknam2001 committed:
fix checkstyle

daknam2001 committed:
add author tag
fix documentation issues

daknam2001 committed:
update PPP

daknam2001 committed:
Merge pull request #307 from daknam2001/editPPP

update PPP

daknam2001 committed:
refactor code to appear slightly more SLAP

daknam2001 committed:
add some assert

daknam2001 committed:
remove redundant assert

daknam2001 committed:
add image for quickstart

hozhenhong99 committed:
Merge pull request #292 from rachelkeh/branch-assessment

Update error message for weightage and marks

hozhenhong99 committed:
Merge pull request #305 from rachelkeh/branch-UG

Update UG to reflect the new 2 dp restriction and justification for w…

hozhenhong99 committed:
Merge pull request #306 from daknam2001/update

add range and selection for delete attendance command

leyondlee committed:
Add allowed characters for values

leyondlee committed:
Minor word change

leyondlee committed:
Change word back

jon-the-melon committed:
Merge pull request #308 from leyondlee/branch-ug

Add allowed characters for values

leyondlee committed:
Add image for empty name edit

leyondlee committed:
Add breaks

jon-the-melon committed:
Merge pull request #309 from leyondlee/branch-ug

Add image for edit_class empty name

jon-the-melon committed:
edit quick notes and sortbyscores to accept maxmarks 0

jon-the-melon committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-3/tp

daknam2001 committed:
add warning for whitespace in userguide in set and delete attendance

jon-the-melon committed:
edit sortbyscores to perform correct action for 0 maxmarks and edit quicknotes

jon-the-melon committed:
Merge pull request #310 from daknam2001/update

add warning for whitespace in userguide in set and delete attendance

hozhenhong99 committed:
Merge pull request #311 from jon-the-melon/master

edit sortbyscores and quicknotes in ug

jon-the-melon committed:
delete random loc in ug and edited kloc for ppps

jon-the-melon committed:
Merge pull request #312 from jon-the-melon/master

delete random loc in ug and edit kloc for ppps

daknam2001 committed:
update usage

hozhenhong99 committed:
update formatting for PPP

hozhenhong99 committed:
Merge branch 'master' of http://github.com/AY2122S1-CS2113T-F12-3/tp into final-edits-branch

daknam2001 committed:
update command summary

jon-the-melon committed:
Merge pull request #313 from daknam2001/update

update usage

jon-the-melon committed:
Merge pull request #314 from daknam2001/update-command-summary

update command summary

jon-the-melon committed:
fix formatting in html for ug command summary

hozhenhong99 committed:
Merge pull request #315 from jon-the-melon/master

fix formatting in html for ug command summary

jon-the-melon committed:
edit set_marks to set_mark in dg

jon-the-melon committed:
Merge pull request #317 from jon-the-melon/master

edit set_marks to set_mark in dg

jon-the-melon committed:
Merge pull request #316 from hozhenhong99/final-edits-branch

update PPP formatting

jon-the-melon committed:
fix html formatting in ug command summary

jon-the-melon committed:
Merge pull request #318 from jon-the-melon/master

fix html formatting in ug command summary

jon-the-melon committed:
change jar file name in quickstart

daknam2001 committed:
Merge pull request #319 from jon-the-melon/master

change jar file name in quickstart

rachelkeh committed:
Update PPP and author tag test class

jon-the-melon committed:
fix set_marks to set_mark in dg and add object diagram for end state

leyondlee committed:
Update TaaClassDiagram and ClassListClassDiagram

jon-the-melon committed:
Merge pull request #320 from rachelkeh/branch-PPP

Update PPP and author tag test class

jon-the-melon committed:
Merge pull request #321 from leyondlee/branch-dg

Update TaaClassDiagram and ClassListClassDiagram

hozhenhong99 committed:
Merge pull request #322 from jon-the-melon/master

fix set_marks to set_mark in dg and add object diagram for end state

hozhenhong99 committed:
add object diagram for add student command

jon-the-melon committed:
add other implementation to marks

jon-the-melon committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-3/tp

hozhenhong99 committed:
update PPP

jon-the-melon committed:
add class and object diagram for other implementation

jon-the-melon committed:
Merge pull request #323 from hozhenhong99/final-edits-branch

add object diagram for add student command

jon-the-melon committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-3/tp

jon-the-melon committed:
edit object diagram to include association

jon-the-melon committed:
Merge pull request #324 from jon-the-melon/master

edit mark section in dg

jon-the-melon committed:
update dg contributions in ppp

jon-the-melon committed:
Merge pull request #325 from jon-the-melon/master

update dg contributions in ppp

daknam2001 committed:
update dg with new sequence diagrams
modify verify in attendance

daknam2001 committed:
update PPP

jon-the-melon committed:
add author

daknam2001 committed:
fix duplicated points in PPP

daknam2001 committed:
Merge pull request #327 from jon-the-melon/master

add author

jon-the-melon committed:
Merge pull request #326 from daknam2001/update-dg

update dg with new sequence diagrams

leyondlee committed:
Use Taa.DECIMAL_PLACES variable

leyondlee committed:
Check for decimal places in deserializers

leyondlee committed:
Add Util.setPrecision method

leyondlee committed:
Add more tests

jon-the-melon committed:
Merge pull request #329 from leyondlee/branch-class

Check for decimal places in deserializers and add more tests

daknam2001 committed:
add header to attendance implementation

daknam2001 committed:
Merge pull request #330 from daknam2001/add-header-to-attendance-implementation

add header to attendance implementation

hozhenhong99 committed:
update DG formatting

daknam2001 committed:
add class diagram

daknam2001 committed:
edit PPP

rachelkeh committed:
Add class diagram for assessment list and update sequence diagram for add assessment by changing class code

rachelkeh committed:
Merge pull request #331 from hozhenhong99/final-edits-branch

update DG formatting

hozhenhong99 committed:
Merge pull request #333 from rachelkeh/branch-DG2

Add class diagram for assessment list and update sequence diagram for…

hozhenhong99 committed:
Merge pull request #332 from daknam2001/class_diagram

add class diagram

daknam2001 committed:
fix minor error in PPP

jon-the-melon committed:
edit object diagram

jon-the-melon committed:
Merge pull request #335 from jon-the-melon/master

edit object diagram

jon-the-melon committed:
Merge pull request #334 from daknam2001/master

fix minor error in PPP
